### ISSUE TITLE
[TECH] Integration tests: licenses compliance

### ISSUE THREAD

**Issue by cirelli94** [ercole-io/ercole#846]
**How should be improved**

Add integration tests on the api `/hosts/technologies/all/databases/licenses-compliance` of api-service.

1. Start ercole, empty, only with oracle licenses types
2. Send some hostdata to data service `/hosts` api
3. Check output of the api in question

**Describe why ercole should be improved in this manner**

To improve reliability of the algorithms.

### Steps
- [ ] @raffaelloferrari  must provide some use cases, with hostdata and expected results
    - One only with some hosts (with and without agreements)
    - One only with some hosts and a vmware cluster  (with and without agreements)
    - One only with some hosts and a veritas cluster (with and without agreements)
    - One only with some hosts and a olvm cluster (with and without agreements)
- [ ] Add this tests (depends on #845)

**Comment by cbiglioli** [comment]
Won't do.
